GPS position
GPS next_goal
float32[3] velocity
float32 heading
float32 battery_SOC

uint32 drone_id
uint64 GPS_timestamp
int8 status

uint8 Run = 1
uint8 Stop = 2
uint8 Land = 3
uint8 Wait = 4
